gig/forms.py

```diff
@@ -130,6 +130,14 @@ def _mergetime(hour, minute='', zone=None):
             self.cleaned_data['has_set_time'] = not set_time is None
             self.cleaned_data['has_end_time'] = not end_time is None
 
+            if not call_time:
+                if set_time:
+                    # if set time is set, but call time is not, default call time to match set time
+                    call_time = set_time
```

Comment:
I think this is OK but I find it uncomfortable. I know when we're playing but I don't care when you show up - why force a call time in there? It might be the right thing to do, just unexpected (for me).

Reply:
The reason I went this way is because many parts of Gig-O expects `date` to be *the* start datetime for the event, and ignore the settime. The most visible places were in the agenda calendar view and the icalendar feeds, but I think there are other places as well. Maybe we could fix this with accessor methods on the model? But I was trying to minimize the amount of extra complication on this already large PR. 

I also would prefer to leave the input as the user intended it. Maybe in a future PR we can fix all the places that depend on `date` and then put this back to being truly optional?